mysql-test/r/mysqld--help-notwin-profiling.result

```diff
@@ -1170,6 +1170,8 @@ The following options may be given as the first argument:
  ON, OFF, FORCE (don't start if the plugin fails to load).
  --rocksdb-deadlock-detect 
  Enables deadlock detection
+ --rocksdb-deadlock-detect-depth=# 
+ Maximum depth a dedlock can have
```

Comment:
dedlock -> deadlock

A better description for users might be:
Number of transactions deadlock detection will traverse through before assuming deadlock.